### PR TITLE
Homework3: Alexander Alexeyuk

### PR DESCRIPTION
**Pull request by AlexAlexeyuk**
### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle)
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks
- [ ] my PR made from the fork feature branch to maxim homework# branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

**Comment by AlexAlexeyuk** [comment]
Hello, please, check  the  file named 'initial commit hw3'

**Comment by AlexAlexeyuk** [comment]
that tests make me crazy...

**Review by belovmd — COMMENTED** [review]
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-summer/blob/master/Homework%20review/Homework3.md):
- created from wrong branch
- 10
- 12
- 28
- 31
- 36

File: HomeWork3/task3.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+a, b, c = 'a', 2, 'python'
+
+
+"""Создайте кортеж из одного элемента, чтобы при итерировании по этому.
+
+ элементы последовательно выводились значения 1, 2, 3. Убедитесь что len()
+ исходного кортежа возвращает 1.
+ """
+tpl = (1, )
```
Comment (belovmd) [inline HomeWork3/task3.py:28]: doesn't solve. previous review not addressed

File: HomeWork3/task4.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+#another one
+
+num = input('Enter few nubmers: ').split()
+dct = {}
+
+for element in num:
+    dct[element] = dct.get(element, 0) + 1
+
+output = sum(map(lambda element: sum(range(1, dct[element])), dct))
```
Comment (belovmd) [inline HomeWork3/task4.py:20]: too complex

**Comment by belovmd** [comment]
not all comments addressed
all CI failed. 
wrong base for branch (tests from hw2 are run)
6

**Pull request closed without merging**